app/src/main/resources/queries/query_for_item.rq

```diff
@@ -10,7 +10,7 @@ SELECT
   (SAMPLE(?wikipediaArticle) AS ?wikipediaArticle)
   (SAMPLE(?commonsArticle) AS ?commonsArticle)
   (SAMPLE(?commonsCategory) AS ?commonsCategory)
-  (SAMPLE(?dateOfOfficialClosure) AS ?dateOfOfficialClosure)
+  (SAMPLE(?DateOfOfficialClosure) AS ?DateOfOfficialClosure)
```

Comment:
Ah, this one was correct. SPARQL variable should start with a lowercase letter. :-)